packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -638,12 +653,15 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
                     // TODO(bkonyi): this won't handle text zoom correctly,
                     //  https://github.com/flutter/flutter/issues/48522
                     height: 56.0,
-                    child: SingleChildScrollView(
-                      dragStartBehavior: widget.dragStartBehavior,
-                      scrollDirection: Axis.horizontal,
-                      reverse: true,
-                      child: Row(
-                        children: footerWidgets,
+                    child: Container(
```

Comment:
A ColoredBox would be better here.

Reply:
Converted, I had to wrap in a SizedBox then do a ColoredBox because you cant modify height.